### PR TITLE
Enabled modern PHP code migration fixers

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Hmmmm, fixing accidental incorrect whitespace characters is again marked as a BC break.

File: src/Faker/Provider/de_CH/Address.php [hunk 1]
```diff
@@ -48,8 +48,8 @@ class Address extends \Faker\Provider\Address
      */
     protected static $canton = [
         ['AG' => 'Aargau'],
-        ['AI' => 'Appenzell Innerrhoden'],
-        ['AR' => 'Appenzell Ausserrhoden'],
+        ['AI' => 'Appenzell Innerrhoden'],
```
Comment (Nyholm) [inline src/Faker/Provider/de_CH/Address.php:51]: What does this change do?

**Review by Nyholm — APPROVED** [review]
Thank you

File: src/Faker/Provider/de_CH/Address.php (hunk 1 above)
Comment (GrahamCampbell) [reply src/Faker/Provider/de_CH/Address.php:51]: Correction of a UTF non-standard whitespace character to a space. I would assume this error got introduced by someone editing the file with the editor using the wrong encoding.

**Pull request merged**